assets/component-menu-drawer.css

```diff
@@ -215,11 +215,27 @@ details[open].menu-opening > .menu-drawer__submenu {
 }
 
 .menu-drawer__utility-links {
-  padding: 2rem;
+  padding: 0;
   background-color: rgba(var(--color-foreground), 0.03);
   position: relative;
 }
 
+.header.header--has-social .menu-drawer__utility-links {
```

Comment:
I don't think any of these rules you have need the additional .header class in the selector here do you?. Those classes shouldn't belong to any other type of element and because we use BEM, it should be clear enough that `header--has-social` is effectively also  targeting `.header`